trin-core/Cargo.toml

```diff
@@ -56,8 +54,7 @@ threadpool = "1.8.1"
 tokio = { version = "1.8.0", features = ["full"] }
 tokio-test = "0.4.2"
 tokio-util = "0.6.8"
-tracing = "0.1.26"
-tracing-futures = "0.2.5"
```

Comment:
I'm surprised that `cargo clippy` or something doesn't tell us when we have completely unused crates, or was this being used somewhere where I'm missing?

Reply:
Unfortunately, `cargo clippy` doesn't handle this for us but there is [cargo udeps](https://github.com/est31/cargo-udeps) which aims to solve this issue.